### PR TITLE
ENG-1332 Adding column wizard enabled to User

### PR DESCRIPTION
**Pull request by BrenoQVDS**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/com/agiletec/aps/system/services/user/IUserManager.java
Comment (alepintus) [inline src/main/java/com/agiletec/aps/system/services/user/IUserManager.java:92]: Hey @BrenoQVDS , 
Could you please translate this javadoc in English ?
Thanks

File: src/main/java/com/agiletec/aps/system/services/user/IUserDAO.java
Comment (alepintus) [inline src/main/java/com/agiletec/aps/system/services/user/IUserDAO.java:118]: Hey @BrenoQVDS , 
Could you please translate this javadoc in English? 
Thanks

**Review by alepintus — CHANGES_REQUESTED** [review]

**Review by ampie — APPROVED** [review]
/LGTM

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/64#pullrequestreview-504553624" title="LGTM">ampie</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)~~ [ampie]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/com/agiletec/aps/system/services/user/IUserManager.java
Comment (BrenoQVDS) [reply src/main/java/com/agiletec/aps/system/services/user/IUserManager.java:92]: I will do that, sorry.
I already changed, but I will commit in my next PR so we don't have to wait for all the tests here, ok?

File: src/main/java/com/agiletec/aps/system/services/user/IUserDAO.java
Comment (BrenoQVDS) [reply src/main/java/com/agiletec/aps/system/services/user/IUserDAO.java:118]: I will do that, sorry.
I already changed, but I will commit in my next PR so we don't have to wait for all the tests here, ok?

**Review by alepintus — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/64#pullrequestreview-504581341" title="Approved">alepintus</a>*, *<a href="https://github.com/entando/entando-engine/pull/64#pullrequestreview-504553624" title="LGTM">ampie</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)~~ [ampie]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
